README.md

```diff
@@ -3,34 +3,24 @@
 ## Usage
 
 ```
-mkdir ~/.s3cfg
-
-docker run -it --rm         \
-  --volume ~/.s3cfg:/.s3cfg \
-  --user $(id -u)           \
-  frew/s3cmd                \
-  configure
-
-docker run -it --rm         \
-  --volume ~/.s3cfg:/.s3cfg \
```

Comment:
I personally feel safer mounting this separately from the rest, but maybe that's silly (if $pwd is ~ it's moot, right?)

Reply:
True...